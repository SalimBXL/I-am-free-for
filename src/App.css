.App {
  background-color: #d4f3d4;
  margin: auto;
  max-width: 1024px;
  max-height: 600px;
  display: flex;
  border-radius: 15px;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  -webkit-box-shadow: 5px 5px 15px 5px #20c0c0; 
  box-shadow: 5px 5px 15px 5px #20c0c0;
  /*padding: 0 1rem 0 1rem;*/
}
