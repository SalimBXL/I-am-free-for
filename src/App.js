import "./styles.css";
import Activities from "./components/Activities/Activities";
import Friends from "./components/Friends/Friends";

export default function App() {
  return (
    <div className="App">
      <h1>I Am Free For...</h1>
      <h2>Start sharing your free time with your friends!</h2>
      <Activities />
      <Friends />
    </div>
  );
}
