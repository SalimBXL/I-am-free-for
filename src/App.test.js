import React from "react";
//import reactDom from "react-dom";
import renderer from "react-test-renderer";
import App from "./App";

it("App Snapshot Test", () => {
    /*
    const div = document.createElement("div");
    ReactDOM.render(<App />, div);
    ReactDOM.unmountComponentAtNode(div);
    */

    const component = renderer.create(<App />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});