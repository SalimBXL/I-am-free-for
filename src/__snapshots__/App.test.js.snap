// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Snapshot Test 1`] = `
<div
  className="App"
>
  <h1>
    I Am Free For...
  </h1>
  <h2>
    Start sharing your free time with your friends!
  </h2>
  <div
    className="Activities"
  >
    <button
      className="Activities-button"
      onClick={[Function]}
      style={null}
    >
      SPORT
    </button>
    <button
      className="Activities-button"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "lightgrey",
          "color": "white",
        }
      }
    >
      COFFEE
    </button>
    <button
      className="Activities-button"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "lightgrey",
          "color": "white",
        }
      }
    >
      RESTAURANT
    </button>
    <button
      className="Activities-button"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "lightgrey",
          "color": "white",
        }
      }
    >
      BARS
    </button>
    <button
      className="Activities-button"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "lightgrey",
          "color": "white",
        }
      }
    >
      CLUB
    </button>
    <button
      className="Activities-button"
      onClick={[Function]}
      style={null}
    >
      CINEMA
    </button>
    <button
      className="Activities-button"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "lightgrey",
          "color": "white",
        }
      }
    >
      NETFLIX
    </button>
    <button
      className="Activities-button"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "lightgrey",
          "color": "white",
        }
      }
    >
      ADULT
    </button>
  </div>
  <div>
    <h3>
      Friends
    </h3>
    <p
      className="Friends-Friend"
    >
      Pseudo#1
       (
      phoneNumber#1
      ) -- 
      sport adult
    </p>
    <p
      className="Friends-Friend"
    >
      Pseudo#2
       (
      phoneNumber#2
      ) -- 
      
    </p>
    <p
      className="Friends-Friend"
    >
      Pseudo#3
       (
      phoneNumber#3
      ) -- 
      adult
    </p>
    <p
      className="Friends-Friend"
    >
      Pseudo#4
       (
      phoneNumber#4
      ) -- 
      coffee restaurant bars club cinema netflix
    </p>
    <p
      className="Friends-Friend"
    >
      Pseudo#5
       (
      phoneNumber#5
      ) -- 
      
    </p>
  </div>
</div>
`;
